package/MDAnalysis/analysis/align.py

```diff
@@ -245,7 +245,8 @@ def rotation_matrix(a, b, weights=None):
     `R` can be used as an argument for
     :meth:`MDAnalysis.core.groups.AtomGroup.rotate` to generate a rotated
     selection, e.g. ::
-
+    >>> A = mda.Universe('topol.tpr','traj.trr')
```

Comment:
use examples from the test files: at the moment it will still fail

Reply:
@lilyminium please jump in if I misunderstood your suggested fix in https://github.com/MDAnalysis/mdanalysis/issues/3365#issuecomment-1082436127

Reply:
@Telomelonia , you can get the test files with
```python
from MDAnalysisTests.datafiles import TPR, TRR
```
and then use `TPR` and `TRR` to create `A`. This makes the example actually executable.